img/README.md

```diff
@@ -0,0 +1,3 @@
+The image `rp.png` is used for tests and must not be changed.
+
+SHA256 (rp.png) = 3b5eeeee7a7326cd6141f54820e6356a0e9d1dd4021407cb1d5e9de9f034ed2f
```

Comment:
I would rather we remove this file and rename `rp.png` to `rp_test_{sha256}.png` along with the hash maybe?

Reply:
Sure, give me a sec. The reason for the README was to clarify that removing the image file might not be a good idea. haha. But since there are not a thousand people working in this project, we should be good.

A file rename coming up...

Reply:
done